specs/protocol/isthmus/exec-engine.md

```diff
@@ -208,17 +212,57 @@ and the `operatorFeeConstant`.
 
 ### Operator Fee
 
-The operator fee, is set as follows:
+The operator fee is integrated directly into the EVM, alongside the standard gas fee and the OP Stack specific L1 data
+fee. This fee must follow the exact same semantics as existing fees charged in the EVM.
 
-`operatorFee = (gasUsed * operatorFeeScalar / 1e6) + operatorFeeConstant`
+#### Fee Formula
+
+$$
+\text{operatorFee} = (\text{gas} \times \text{operatorFeeScalar} \div 10^6) + \text{operatorFeeConstant}
+$$
 
 Where:
 
-- `gasUsed` is amount of gas used by the transaction.
+- `gas` is the amount of gas that the transaction used. When calculating the amount of gas that is bought at the
```

Comment:
Are you defining `gas` generally or as a specific term in this context?

Reply:
Generally. Semantically, the value that's passed into this function isn't necessarily the actual gas used post-execution. Pre-execution, we always assume that the gas cost will be the worst-case (using `gas_limit` when calculating it.)